lightning/src/chain/channelmonitor.rs

```diff
@@ -829,7 +824,8 @@ pub(crate) struct ChannelMonitorImpl<Signer: WriteableEcdsaChannelSigner> {
 	// we further MUST NOT generate events during block/transaction-disconnection.
 	pending_monitor_events: Vec<MonitorEvent>,
 
-	pending_events: Vec<Event>,
+	pub(super) pending_events: Vec<Event>,
+	pub(super) pending_events_processor: bool,
```

Comment:
Could we name this something like `is_processing_pending_events`. The _processor_ suffix is a little confusing.